ext/crypto/lib.rs

```diff
@@ -272,12 +273,21 @@ pub enum KeyFormat {
   Pkcs8,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum KeyType {
+  Private,
+  Public,
+  Secret,
+}
+
```

Comment:
```suggestion
use crate::key::KeyType;
```
It's already defined in `ext/crypto/key.rs` ^_^